packages/tame-metering/src/tame.js

```diff
@@ -242,10 +242,7 @@ export function tameMetering() {
         try {
           defineProperty(o, p, newDesc);
         } catch (e) {
-          // Ignore: TypeError: Cannot redefine property: Symbol(Symbol.hasInstance)
-          if (typeof p !== 'symbol' || !(e instanceof TypeError)) {
-            throw e;
-          }
+          // Ignore: TypeError: Cannot redefine property: ...
```

Comment:
Should this log the failure?

Reply:
This is best-effort code only, and logging would be annoying for no real gain.

Reply:
As requested, I've added logging.